### ISSUE TITLE
Track vim9 development

### ISSUE THREAD

**Issue by myitcv** [govim/govim#1086]
Supporting Vim9 will apparently give lots of performance improvements, per https://groups.google.com/g/vim_announce/c/tfp72VXQuZc:

https://github.com/vim/vim/blob/master/runtime/doc/vim9.txt

I suspect we will need to dual-maintain support for Vim8 and Vim9 for some time, because Vim9 script is not backwards compatible. 

But nothing to do for now until the development settles down (which could still be some months, per the update from Bram)

**Comment by myitcv** [comment]
FYI @leitzler

**Comment by ehdis** [comment]
Distributions are starting packaging vim9. Does this plugin works out of the box with the new major version of vim? Thanks!

**Comment by myitcv** [comment]
> Distributions are starting packaging vim9. Does this plugin works out of the box with the new major version of vim? Thanks!

Yes it does. We haven't rewritten the `govim` scripts in Vim9 script, but that's not a blocker to using `govim` with Vim9 (as I am)
